recipe/build.sh

```diff
@@ -33,6 +33,7 @@ if [[ "${target_platform}" == linux-aarch64 ]]; then
   BINARY="64"
 elif [[ "${target_platform}" == linux-ppc64le ]]; then
   TARGET="POWER8"
+  BUILD_BFLOAT16=1
```

Comment:
Can you enable this for `x86_64` platforms as well?

Reply:
This would depend on which x86 processors are supporting bfloat16.  Further, we have a basic understanding (team mandate, if you will) that it's inappropriate to make changes that will affect behavior on other architectures in any case, particularly when we have no way to test those changes.

Reply:
Please understand, I'm personally rather new to the open source world, and so I apologize if my answer seems strident...

Reply:
Are there any action items remaining for me, before this can be merged?

Reply:
One question. Does this mean that processors without `bfloat16` support would get crashes? If not, let's enable it for x86_64. If yes, I don't think we can merge this.

Reply:
Please note that the one line change I made ONLY affects processors in the Power family.  
It does not change ANYTHING that would ever affect an x86-64 machine.
This is by design.
I cannot imagine how a non-Power-family processor could ever crash if we add this change.

However, I do NOT want to insert any code that would affect machines we aren't equipped to test on.
I would view that as dangerous and careless behavior.

It should be easy for anyone who is invested in x86-64 machines to mimic our change after it is adopted, 
and do their own testing, right?

Reply:
@isuruf Considering the fact that there is bfloat16 optimization for some of level 1/2 functions in openblas for x86, this should work for x86_64.  However, I am not sure about other architectures. 
In the future, we also consider enabling this option by default in openblas for POWER.